scijava/scijava-ops-engine/src/test/java/org/scijava/ops/engine/hints/AdaptationHintTest.java

```diff
@@ -81,8 +79,7 @@ public void testNonAdaptableOp() {
 			ops.op("test.adaptation.unadaptable").inType(Double[].class).outType(
 				Double[].class).computer();
 			throw new IllegalStateException("This op call should not match!");
-		} catch( IllegalArgumentException e) {
-			assertTrue(e.getCause() instanceof OpMatchingException);
+		} catch( OpMatchingException e) {

```

Comment:
Instead of an empty catch block, there should be a comment here saying something like:
```java
// NB: Expected behavior.
```

Reply:
scijava/scijava#84

Reply:
Fixed this one in 98850ae3. We can catch (hah) the rest of them in a dedicated response to that issue.